common/utils/src/main/resources/error/error-classes.json

```diff
@@ -2333,6 +2354,11 @@
     ],
     "sqlState" : "428C4"
   },
+  "UNRECOGNIZED_PARAMETER_NAME" : {
+    "message" : [
+      "Cannot invoke function <functionName> because the function call included a named argument reference for the argument named <argumentName>, but this function does not include any signature containing an argument with this name."
```

Comment:
Could you help users and propose function parameters sorted by levenshtein distance? See for instance:
https://github.com/apache/spark/blob/d9c2d38f14a400ea078492d770b749f506a8401c/sql/catalyst/src/main/scala/org/apache/spark/sql/catalyst/util/StringUtils.scala#L80

Reply:
Sounds good! Added the feature.